src/explore-education-statistics-common/src/modules/find-statistics/components/KeyStat.tsx

```diff
@@ -73,6 +73,11 @@ const KeyStat = ({
               <Details
                 summary={summary?.dataDefinitionTitle[0] || 'Help'}
                 className={styles.definition}
+                visuallyHiddenText={
```

Comment:
Don't think this is quite right as it will only render the hidden text if an analyst has set the `dataDefinitionTitle` to 'Help', which is unlikely.  
I think it should render when the `dataDefinitionTitle` isn't set as that's when the visible text defaults to 'Help':
```
visuallyHiddenText={
 !summary?.dataDefinitionTitle[0]'
 ? `for ${keyStat.title}`
 : undefined
  }
```

Reply:
Ah yeah, you're right. Think I got this one mixed up